config/routes.rb

```diff
@@ -91,9 +91,15 @@
 #                                                                        PUT    /api/v2/access_requests/:id(.:format)                                                                                                    api/v2/access_requests#update
 #                                                                        DELETE /api/v2/access_requests/:id(.:format)                                                                                                    api/v2/access_requests#destroy
 #                                                api_v2_build_new_course GET    /api/v2/build_new_course(.:format)                                                                                                       api/v2/courses#build_new
+#                                       api_v3_recruitment_cycle_courses GET    /api/v3/recruitment_cycles/:recruitment_cycle_year/courses(.:format)                                                                     api/v3/course_searches#index
```

Comment:
I think there we should expose `/course` and use that as the default for frontend

`recruitment_cycle_year` is nice but I don't think we have an actual need to expose it.

Reply:
Ya, I think that that makes sense too. `find` doesn't need to know what the current cycle is.